### ISSUE TITLE
Add title screen

### ISSUE THREAD

**Issue by alexdevteam** [tccppgg/rect#27]
Add a simple title screen that has the following elements:

- [ ] A start game button that transitions to another layout.
- [ ] A title element (Maybe the rect sign in objects.png?)
- [ ] An exit button that closes the window

This issue depends on #25. Please don't start working on the systems/components without codegen, as a rewrite would be needed.

Settings, credits and whatnot have low priority as we can use files along the executable for now.
